fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java

```diff
@@ -703,71 +721,27 @@ public PlanFragment visitPhysicalHudiScan(OptExpression optExpression, ExecPlan
         @Override
         public PlanFragment visitPhysicalHiveScan(OptExpression optExpression, ExecPlan context) {
             PhysicalHiveScanOperator node = (PhysicalHiveScanOperator) optExpression.getOp();
+            ScanOperatorPredicates predicts = node.getScanOperatorPredicates();
```

Comment:
Better use full name like `predicates`. `Predicts` and `Predicates` are two different words.

Reply:
done